src/feature-lifecycle/summary.md

```diff
@@ -1,3 +1,51 @@
 # The feature lifecycle
 
-**Status:** Stub
+## Identifying the problem
+
+The first step before proposing any change to the standard library is to properly identify the problem that is trying to be solved. This helps to identify cases of the [XY problem] where a better solution exists without needing any changes to the standard library.
+
+For this reason it is helpful to focus on real problems that people are encountering, rather than theoretical concerns about API design.
+
+[XY problem]: https://en.wikipedia.org/wiki/XY_problem
+
+## Suitability for the standard library
+
+Unlike third party crates on crates.io, the Rust standard library is not versioned. This means that any stable API that is added can *never* be removed or modified in a backwards-incompatible way. For this reason, the standard library maintainers place a high bar on any change to the standard library API.
+
+APIs that are well suited to the standard library are things that require language and/or compiler support, or that extend existing standard library types. Complex APIs that are expected to evolve over time (e.g. GUI frameworks) are a poor fit due to the lack of versioning.
+
+The API Change Proposal process is intended to be a lightweight first step to
+getting new APIs added to the standard library. The goal of this process is to
+make sure proposed API changes have the best chance of success. The ACP process
+accomplishes this by ensuring all changes are reviewed by the library API team,
+who will evaluate the proposal and accept it if they are optimistic that the proposal will
+be merged and pass its eventual FCP.
+
+You can create an ACP in the `rust-lang/libs-team` repo using [this issue template](https://github.com/rust-lang/libs-team/issues/new?assignees=&labels=api-change-proposal%2C+T-libs-api&template=api-change-proposal.md&title=%28My+API+Change+Proposal%29). This should include a sketch of the proposed API, but does not have to be the final design that will be implemented.
+
+## API design exploration
+
+Once a feature is deemed suitable for inclusion in the standard library, the exact design should be iterated on to find the best way to express it as a Rust API. This iteration should happen in community forums such as [Rust internals](https://internals.rust-lang.org/) where all members of the community can comment and propose improvements.
+
+Keep the following points in mind during the discussion:
+- Resist the temptation to make the API overly general since this tends to increase complexity and lowers the chance of it being accepted by the library API team. Instead, try to stay focused on the problem that needs solving.
```

Comment:
This needs more careful phrasing we often do ask "where does this lead" and "are there more general options".